@kiva/kv-components/vue/KvTextLink.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+	<component
+		:is="tag"
+		:to="to"
+		:href="href"
+		:disabled="state === 'disabled'"
+		class="
+			tw-text-h4
+			tw-text-action-700 hover:tw-text-action-700 hover:tw-underline
+			tw-inline-flex tw-items-center tw-gap-x-0.5
+		"
+		:class="{
+			'tw-opacity-low tw-pointer-events-none': state === 'disabled',
+		}"
+		@click="onClick"
+	>
+		<slot></slot>
+		<kv-material-icon
```

Comment:
Could we also do a before icon... or toggle to change it's position between beginning and end?

Reply:
This is something we can add later when the need arises.